oss_src/process/process_unix.cpp

```diff
@@ -32,17 +33,21 @@ namespace fs = boost::filesystem;
  * handler.
  */
 graphlab::mutex sigchld_handler_lock;
-std::set<size_t> proc_ids_to_reap;
+
```

Comment:
prefer std::call_once rather than pthread_once. It is somewhat more portable.
See oss_src/fileio/block_cache.cpp for an example